### PR TITLE
Fix pdf export for the georgian language

### PR DESCRIPTION
**Pull request by lunars97**
### Short description

Currently the export function is not showing correct georgian characters, instead it is showing boxes. 

### Proposed changes

- Add fonts for georgian language
- Make necessary adjustments in other components so that georgian language is supported in various browsers and it is versions

### Side effects

- Should be none

### Resolved issues

Fixes: #2857 

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

**Comment by lunars97** [comment]
@MizukiTemma do you happen to know what is the problem with the testing failures? :pray:  I have really no idea :thinking:

**Comment by MizukiTemma** [comment]
> @MizukiTemma do you happen to know what is the problem with the testing failures? 🙏 I have really no idea 🤔

You can probably remove the new language object from the test data 👀

**Comment by lunars97** [comment]
> > @MizukiTemma do you happen to know what is the problem with the testing failures? 🙏 I have really no idea 🤔
> 
> You can probably remove the new language object from the test data 👀

I also think about, but I thought that maybe it is because of the font or sth like that. I will try to remove it and see :) Thank you a lot for getting back to me :heart_hands:

**Review by MizukiTemma — APPROVED** [review]
Looks good 😸 Thank you!

File: integreat_cms/release_notes/current/unreleased/2857.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+en: Fix PDF export function for the georgian language
+de: Behebe der PDF-Exportfunktion für die georgische Sprache
```
Comment (JoeyStk) [inline integreat_cms/release_notes/current/unreleased/2857.yml:2]:
```suggestion
de: Behebe die Probleme bei der PDF-Exportfunktion für Georgisch
```

**Review by JoeyStk — APPROVED** [review]
Thank you very much :blush: 
I have a small comment on the release note, but other than that it looks good :) 
After you squashed the commits, I think this PR is good to go :rocket:

**Pull request merged**